CMakeLists.txt

```diff
@@ -956,6 +956,9 @@ IF(REDUCE_SWIG_DEBUG)
 	SET(SWIG_CXX_COMPILER_FLAGS "${SWIG_CXX_COMPILER_FLAGS} -g1")
 ENDIF(REDUCE_SWIG_DEBUG)
 
+# For Swig and Unit-tests
+FIND_PACKAGE(Jinja2)
```

Comment:
this just wrong... why to test it when you dont compile the tests or the modular interfaces?